.github/workflows/job_openvino_js.yml

```diff
@@ -46,19 +45,18 @@ jobs:
       # Needed as ${{ github.workspace }} is not working correctly when using Docker
       - name: Setup Variables
         run: |
-          echo "OPENVINO_JS_DIR=$GITHUB_WORKSPACE/openvino/src/bindings/js" >> "$GITHUB_ENV"
-          echo "OPENVINO_JS_LIBS_DIR=$GITHUB_WORKSPACE/openvino/src/bindings/js/node/bin" >> "$GITHUB_ENV"
+          echo "OPENVINO_JS_DIR=$GITHUB_WORKSPACE/openvino/src/bindings/js/node" >> "$GITHUB_ENV"
 
       - name: Download OpenVINO artifacts (JS)
         uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e # v4.2.1
         with:
-          pattern: openvino_[js]*
-          path: ${{ env.OPENVINO_JS_LIBS_DIR }}
+          name: openvino_js_package
+          path: ${{ env.OPENVINO_JS_DIR }}
           merge-multiple: true
 
       - name: Extract OpenVINO packages
-        run: pigz -dc openvino_js_package.tar.gz | tar -xf - -C ${OPENVINO_JS_LIBS_DIR}
-        working-directory: ${{ env.OPENVINO_JS_LIBS_DIR }}
+        run: tar -xzf openvino_js_package.tar.gz -C ${OPENVINO_JS_DIR}
```

Comment:
Why not use `pigz`? It should be faster, cc @mryzhov.

Reply:
I revert these changes and create a separate packing for npm package artifacts